WashU-IT-RIS/coldfront-wustl-fork
# Reset ACLs feature (ITDEV-35098)

Testing instructions:

- Build `dev` and `worker` images from this branch and deploy locally or to QA.
- Create allocation
- Optionally, create one or more sub-allocations under initial allocation
- Add files and directories as desired, optionally "corrupting" their ACLs
- Navigate to "View All Allocations" and select one of the created allocations
  - click red "Reset ACLs" button, optionally selecting "Include sub-allocations" checkbox if applicable
- Observe successful job completion in admin console.
